siliconcompiler/schema/schema_obj.py

```diff
@@ -460,8 +490,15 @@ def error_msg(t):
         raise ValueError(f'Invalid field {field} for keypath {keypath}')
 
     @staticmethod
-    def _is_empty(cfg):
-        return not cfg['set']
+    def _is_set(cfg, step=None, index=None):
```

Comment:
If I'm reading this method correctly, it is supposed to indicate whether a parameter has been set to a value which is different from its default configuraiton?

Even though it's an internal method, it might be a good idea to add a docstring so that people don't assume `_is_set` means "has a non-empty value".

Reply:
It indicates whether a parameter has been set to any value by a user explicitly. Good suggestion - I added a docstring!